test/spec/api_helpers.rb

```diff
@@ -53,7 +55,15 @@ def api_create_team_project_and_claim(quit = false, quote = 'Claim')
     data = api_create_team_and_project
     claim = request_api 'claim', { quote: quote, email: data[:user].email, team_id: data[:team].dbid, project_id: data[:project].dbid }
     @driver.quit if quit
-    claim
+    claim  
+  end
+
+  def api_create_team_and_project
```

Comment:
Why this method duplicates the one above? They have the same name

Reply:
One was old, with not accepting team name as parameter. Deleted.